### PR TITLE
Fix unit tests and make sure all unit tests pass

### PR DESCRIPTION
**Pull request by nickyfantasy**

### CODE REVIEW

File: visualdl/logic/sdk_test.cc [hunk 1]
```diff
@@ -132,40 +129,6 @@ TEST(Image, add_sample_test) {
   CHECK_EQ(image2read.num_records(), num_steps);
 }
 
-TEST(Image, add_sample_test) {
```
Comment (daming-lu) [inline visualdl/logic/sdk_test.cc:135]: So we do not test Image?

File: visualdl/logic/sdk_test.cc [hunk 2]
```diff
@@ -180,14 +143,14 @@ TEST(Audio, test) {
   for (int step = 0; step < num_steps; step++) {
     audio.StartSampling();
     for (int i = 0; i < 7; i++) {
-      vector<int64_t> shape({5, 5, 3});
-      vector<float> data;
-      for (int j = 0; j < 3 * 5 * 5; j++) {
-        data.push_back(float(rand()) / RAND_MAX);
+      vector<int32_t> shape({16000, 2, 2});
```
Comment (daming-lu) [inline visualdl/logic/sdk_test.cc:146]: What is the reason of changing these numbers? What are their meanings? Just want to learn 😄

File: visualdl/logic/sdk_test.cc [hunk 3]
```diff
@@ -217,9 +180,9 @@ TEST(Audio, add_sample_test) {
   for (int step = 0; step < num_steps; step++) {
     audio.StartSampling();
     for (int i = 0; i < 7; i++) {
-      vector<float> data;
-      for (int j = 0; j < 3 * 5 * 5; j++) {
-        data.push_back(float(rand()) / RAND_MAX);
+      vector<uint8_t> data;
```
Comment (daming-lu) [inline visualdl/logic/sdk_test.cc:183]: Also here 😃

File: visualdl/logic/sdk_test.cc (hunk 1 above)
Comment (nickyfantasy) [reply visualdl/logic/sdk_test.cc:135]: we do, its duplicated, add_sample_test already defined

File: visualdl/logic/sdk_test.cc (hunk 3 above)
Comment (nickyfantasy) [reply visualdl/logic/sdk_test.cc:183]: The audio data we accept is vector of uint8_t type, so I randomly pick numbers from rand() % 256 to create a vector

File: visualdl/logic/sdk_test.cc (hunk 2 above)
Comment (nickyfantasy) [reply visualdl/logic/sdk_test.cc:146]: last time I just copied image test and change to audio. Image test they use shape [5,5,3] to represent 5x5 h and w and 3 channels (RGB). In audio, the params are [sample_rate, sample_width, number of channels]

**Review by daming-lu — APPROVED** [review]

**Pull request merged**